src/helpers/tests/dimensions.spec.ts

```diff
@@ -260,6 +260,97 @@ describe('generateDimensions', () => {
                 .toEqual(expectedDimensions);
         });
     });
+    describe('area chart', () => {
+        it('should generate dimensions for area with one measure', () => {
+            const expectedDimensions: AFM.IDimension[] = [
+                {
+                    itemIdentifiers: ['measureGroup']
+                },
+                {
+                    itemIdentifiers: []
+                }
+            ];
+
+            expect(generateDimensions(getVisualization('onemeasure'), VisualizationTypes.AREA))
```

Comment:
imho, it would be nice with
```
const dimensions = generateDimensions(getVisualization('onemeasure'), VisualizationTypes.AREA);
expect(dimensions).toEqual(expectedDimensions);
```